### ISSUE TITLE
Fail to Install FlareVM

### ISSUE THREAD

**Issue by adimenia** [fireeye/flare-vm#96]
Hello
 when trying to install FlareVM using the following link: http://boxstarter.org/package/url?https://raw.githubusercontent.com/fireeye/flare-vm/master/flarevm_malware.ps1 im getting the following error
![image](https://user-images.githubusercontent.com/5006200/40584876-a2ef9634-61b1-11e8-9915-f06909eabd3f.png)
the attached log provides a full description of the error.
[CAAW4RIB.log](https://github.com/fireeye/flare-vm/files/2042491/CAAW4RIB.log)

**Comment by pleedom** [comment]
I read somewhere that the web installer was deprecated, but the powershell script is failing as well. Execution policy was set, running in an elevated powershell session. 
```  
[ * ] Installing Boxstarter
Exception calling "DownloadString" with "1" argument(s): "The underlying connection was closed: Could not establish tru
st relationship for the SSL/TLS secure channel."
At C:\Users\user\Desktop\flare-vm-master\flare-vm-master\install.ps1:30 char:56
+   iex ((New-Object System.Net.WebClient).DownloadString <<<< ('http://boxstarter.org/bootstrapper.ps1')); get-boxstar
ter -Force
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : DotNetMethodException

The term 'get-boxstarter' is not recognized as the name of a cmdlet, function, script file, or operable program. Check
the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\user\Desktop\flare-vm-master\flare-vm-master\install.ps1:30 char:115
+   iex ((New-Object System.Net.WebClient).DownloadString('http://boxstarter.org/bootstrapper.ps1')); get-boxstarter <<
<<  -Force
    + CategoryInfo          : ObjectNotFound: (get-boxstarter:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

[ * ] Getting user credentials ...

Windows PowerShell Credential Request
Enter your credentials.
Password for user user:

The term 'Install-BoxstarterPackage' is not recognized as the name of a cmdlet, function, script file, or operable prog
ram. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At C:\Users\user\Desktop\flare-vm-master\flare-vm-master\install.ps1:39 char:32
+       Install-BoxstarterPackage <<<<  -PackageName https://raw.githubusercontent.com/fireeye/flare-vm/master/flarevm_
malware.ps1 -Credential $cred
    + CategoryInfo          : ObjectNotFound: (Install-BoxstarterPackage:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

**Comment by PJF16** [comment]
Just wanted to install Flare with the install1.ps script. I get the same error as @pleedom.

**Comment by hashem-sadeddin** [comment]
@pleedom @PJF16 I had the same issue and resolved it.

If I were you, I would make sure you are on Windows 7 SP1. I didn't have the service pack installed in my VM and I believe that caused the issue.

If that doesn't resolve it for you, then download the bootstrapper.ps1 script manually and change the install.ps1 script to import the module as a file:

`$ScriptPath = Split-Path $MyInvocation.InvocationName
  Import-Module -Force "$ScriptPath\bootstrapper.ps1"; get-boxstarter -Force`

I actually did the latter step first, but I'm pretty sure all you actually need is SP1. In case I'm wrong, try the last step as well as making sure you have SP1.

**Comment by petermat** [comment]
As error says, certificate is not trusted. Solution is to install comodo certificate and than it works like a charm. Google -> "Install comodo root CA"

**Comment by DynaMc** [comment]
I just wanted to add that I ran into this same error and adding the comodo cert worked for me, as @petermat suggested.   

The before and after of adding the cert.

![image](https://user-images.githubusercontent.com/5316899/42734110-51658b26-880c-11e8-85ef-bb25c690ca64.png)

**Comment by kobs0N** [comment]
> 
> 
> As error says, certificate is not trusted. Solution is to install comodo certificate and than it works like a charm. Google -> "Install comodo root CA"

Works like a charm!!! Thanks
I used that link https://support.comodo.com/index.php?/comodo/Knowledgebase/Article/View/977/108/extended-validation-sha-2

**Comment by htnhan** [comment]
Should be fixed in 237f87ecb516e06a47a21db00743c369119caf1f
